### DISCUSSION TITLE
feat: typecho 迁移 halo

### DISCUSSION THREAD

**Discussion by wjuxiao** [halo-dev/halo#4792] [Ideas]
## 概述

  typecho 迁移 halo 工具或插件

## 动机

  早先没有发现比较好用的java开源blog。如今发现halo堪称完美，但是个人博客开得比较早，用了typecho，文章数量比较多，手动迁移过于繁琐。且我相信，应该也有蛮多用户有类似的需求，因此希望能有 迁移工具或插件

**Comment by ruibaby** [comment]
之前是有这个计划的，但是看了下 Typecho 导出的数据被混淆过或者加密过，就没有继续弄了，而且目前 Halo 没有插件机制（已有计划），后面会研究一下迁移方案。

**Comment by JohnNiang** [comment]
Hi @ruibaby ，目前 Halo 2.x 已经有插件机制了，是否可以考虑重新打开当前 Issue 呢？

[a comment by an automated account is not shown]

**Comment by ruibaby** [comment]
可以等待 https://github.com/halo-sigs/plugin-migrate/issues/2 完成。
